Warehouse doc updates (#2533)
segmentio/segment-docs: src/connections/storage/warehouses/faq.md

If you want to insert custom data into your warehouse, create new schemas that are not associated with an existing source, since these may be deleted upon a reload of the Segment data in the cluster.

> Segment highly recommends scripting any sort of additions of data you might have to warehouse, so that you aren't doing one-off tasks that can be hard to recover from in the future in the case of hardware failure.

Reviewer: ```suggestion
Segment recommends scripting any sort of additions of data you might have to warehouse, so that you aren't doing one-off tasks that can be hard to recover from in the future in the case of hardware failure.
```

## Which IPs should I allowlist?

You can allowlist Segment's custom IP `52.25.130.38/32` while authorizing Segment to write in to your Redshift or Postgres port.

If you're in the EU region, use CIDR `3.251.148.96/29`.

> info ""
> EU workspace regions are currently in beta. If you would like to learn more about the beta, please contact your account manager.

BigQuery does not require allowlisting an IP address. To learn how to set up BigQuery, check out Segment's BigQuery [set up guide](/docs/connections/storage/catalog/bigquery/#getting-started)

## Will Segment sync my historical data?

> Segment will automatically load up to two months of your historical data when you connect a warehouse.

Reviewer: ```suggestion
Segment loads up to two months of your historical data when you connect a warehouse.
```

- **Config API**: Send a [PATCH Connected Warehouse request](https://reference.segmentapis.com/?version=latest#ec12dae0-1a3e-4bd0-bf1c-840f43537ee2) to update the settings for the warehouse(s) you want to prevent from syncing.

After a source is created, you can enable or disable a warehouse sync within the Warehouse Settings page.

## Can I be notified when warehouse syncs fail?

> If you enabled activity notifications for your storage destination, you will receive notifications in the Segment app when your warehouse syncs fail.

Reviewer: ```suggestion
If you enabled activity notifications for your storage destination, you'll receive notifications in the Segment app when warehouse syncs fail.
```

- **Config API**: Send a [PATCH Connected Warehouse request](https://reference.segmentapis.com/?version=latest#ec12dae0-1a3e-4bd0-bf1c-840f43537ee2) to update the settings for the warehouse(s) you want to prevent from syncing.

After a source is created, you can enable or disable a warehouse sync within the Warehouse Settings page.

> ## Can I be notified on warehouse sync failures?

Reviewer 2: Can we please add here that customers that sign up for notifications in the Notifications tab in app will receive notifications for partial and full failures on the 5th and 20th consecutive failures? Thanks

> ## How is my data formatted in my warehouse?

Reviewer 2: Can we please mention that data will always be snake cased? Thanks!

## If my syncs fail and get fixed, will I need to ask for a backfill?

> Yes! If your syncs fail, you will need to ask for a backfill, as this is a manual process. 

Reviewer 2: This is incorrect. Our warehouse syncs are incremental, and we'll keep a copy of the data coming through the customer's source in our own internal buckets. Once a successful sync takes place again, we'll automatically load all of the missed data since the last successful sync, so there's no need for a backfill.

> ## Can I change my schema names once they've been created?

Reviewer 2: We can link this to the section where we talk about tweaking things on the warehouses side on the doc. From within the Segment app, we'll store schema names under each source's SQL Settings, and changing the name there will cause the schema to be split into two from that moment onwards. In order to change a schema name with no disruptions, these are the steps we would recommend:

Disable the warehouse in the Segment UI: Warehouse > Settings > Basic
Rename the schema in Segment in your source settings: Sources > Settings > SQL Settings
Rename the schema in your actual warehouse to the new name
Re-enable the warehouse

Thanks!
